db/migrations/20170627214757_initial.js

```diff
@@ -0,0 +1,28 @@
+
+exports.up = (knex, Promise) => {
+  return Promise.all([
+    knex.schema.createTable('folders', (table) => {
+      table.increments('id').primary()
+      table.string('folder_name')
+
+      table.timestamps(true, true)
+    }),
+
+    knex.schema.createTable('urls', (table) => {
```

Comment:
This looks wooooooonnnnnddddddeeeeeeeerrrrrrrrrful! I think you did a great job setting up our knex and db. It is simple and concise and covers that sections we will be using the most, so nice job!